js/src/dapps/tokenreg/Tokens/actions.js

```diff
@@ -98,13 +105,33 @@ export const loadToken = (index) => (dispatch, getState) => {
         base: result[2].toNumber(),
         name: result[3],
         owner: tokenOwner,
+        ownerAccountInfo: accountsInfo[tokenOwner],
         isPending: false,
         isTokenOwner
       };
 
+      return data;
+    })
+    .then(data => {
+      return getTokenTotalSupply(data.address)
+        .then(totalSupply => {
+          data.totalSupply = totalSupply;
+          return data;
+        });
+    })
+    .then(data => {
+      // If no total supply, must not be a proper token
+      if (data.totalSupply === null) {
```

Comment:
does it come back at null or 0? May want to do a eth_getCode on the address first, no code found, not a contract